SwiftBar/AppDelegate.swift

```diff
@@ -125,17 +125,23 @@ class AppDelegate: NSObject, NSApplicationDelegate, SPUStandardUserDriverDelegat
                                                title: url.queryParameters?["title"],
                                                subtitle: url.queryParameters?["subtitle"],
                                                body: url.queryParameters?["body"],
+                                               open: url.queryParameters?["open"],
```

Comment:
I foresee other actions, like expand menubar item or run a script. Therefore I suggest to name this one "href", it will be aligned with "href" in menu item parameters

Reply:
Changed